css/design_system/index.scss

```diff
@@ -2,6 +2,7 @@
 @import "tokens/spacing";
 @import "tokens/shadows";
 
+@import "content";
```

Comment:
Sorry didn't notice this earlier - this should probably be imported _after_ we define the paragraph and link element styles (i.e. on line 15).